lib/core/src/lib/form/components/widgets/checkbox/checkbox.widget.html

```diff
@@ -6,10 +6,8 @@
                   [disabled]="field.readOnly || readOnly"
                   [(ngModel)]="field.value"
                   (ngModelChange)="onFieldChanged(field)"
-                  [matTooltip]="field.tooltip"
-                  (click)="markAsTouched()"
-                  matTooltipPosition="right"
```

Comment:
Same here for position

Reply:
the position should not be there, it's the browser who decides on positioning (i.e. scrolling, RTL layout, etc)